src/UI/WarehouseInspector.cpp

```diff
@@ -82,19 +82,25 @@ void WarehouseInspector::update()
 
 	Window::update();
 
-	Renderer& r = Utility<Renderer>::get();
+	Renderer& renderer = Utility<Renderer>::get();
 
-	ProductPool& _pool = mWarehouse->products();
+	ProductPool& pool = mWarehouse->products();
 
-	r.drawText(*FONT_BOLD, "Storage", rect().x() + constants::MARGIN, rect().y() + 25.0f, 255, 255, 255);
-	r.drawText(*FONT, string_format("%i / %i", _pool.availableStorage(), _pool.capacity()), rect().x() + constants::MARGIN + FONT_BOLD->width("Storage") + 20, rect().y() + 25.0f, 255, 255, 255);
+	const auto drawTitleText = [&renderer](NAS2D::Point<int> position, std::string title, std::string text, int offset = 100) {
```

Comment:
Consider changing the `std::string` parameter types to `const std::string&` or `std::string_view` because they don't change within the function.